maven-release-manager/src/main/java/org/apache/maven/shared/release/DefaultReleaseManagerListener.java

```diff
@@ -86,16 +86,9 @@ public void goalStart( String goal, List<String> phases )
 
     public void phaseStart( String name )
     {
-        if ( goal == null || ( ( currentPhase + 1 ) >= phases.size() ) )
```

Comment:
Why is this information not necessary anymore?

Reply:
this code that had to take care of "phases" not in a goal is the proof that it was not a phase :)

Reply:
Right.